docs/conf.py

```diff
@@ -52,9 +56,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '2.0+git'
+version = ZULIP_VERSION
```

Comment:
So, this is a semantic change from e.g. `2.0` to `2.0.7`.  Have you found the docs for what these two actually do?  I had thought there was a reason for these two values to be different, but I don't actually know what it is so it's possible there isn't.

Reply:
Sphinx upstream [says](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version): “If you don’t need the separation provided between `version` and `release`, just set them both to the same value.” As far as I can tell, their actual interpretation is up to the theme, and the ReadTheDocs theme only [uses](https://github.com/readthedocs/sphinx_rtd_theme/blob/0.4.3/sphinx_rtd_theme/layout.html#L44) `release`.

Reply:
Amazing.  I guess their `X.Y` comment there probably created the confusion resulting in this.

It's potentially the case that our docs might be happier if we just documented the latest 2.0.x, rather than every point release, but we'd need to change the theme to do that, so it's not actually relevant to this question.

Reply:
Well that comment came from Sphinx upstream via `sphinx-quickstart`, not from RTD.